### PR TITLE
Redesigned site footer

### PR DESCRIPTION
**Pull request by ahmad-shahid**

### CODE REVIEW

File: src/gcweb/footer.ejs [hunk 1]
```diff
@@ -4,46 +4,91 @@
...
-                    <%_ if (pr.showFooter != false) { -%>
-                        <ul>
+    <h2 class="wb-inv">About this site</h2>
+    <%_ if (pr.contextualFooter != null) { -%>
+        <div class="gc-contextual">
+            <div class="container">
+                <nav>
+                    <h3><%- pr.contextualFooter.title %></h3>
+                    <ul class="list-col-xs-1 list-col-sm-2 list-col-md-3">
+                        <%_ for (var footerIndex=0; footerIndex<(pr.contextualFooter.link.length > 3 ? 3 : pr.contextualFooter.link.length); footerIndex++) {
```
Comment (plup2) [inline src/gcweb/footer.ejs:19]: Two things here:
- `contextualFooter.link` should be check to make sure it is not null/undefined before doing the for
- `contextualFooter.link` should be named `contextualFooter.links`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # CDTS Change Log
```
Comment (plup2) [inline CHANGELOG.md:1]: Also change version in package.json 4.0.47

**Review by plup2 — CHANGES_REQUESTED** [review]

File: src/gcweb/footer.ejs [hunk 3]
```diff
@@ -4,46 +4,91 @@
  * @param? termsLink Used to override the Terms and conditions link in the footer.
  * @param? privacyLink Used to override the Privacy link in the footer.
  * @param? showFooter Used to determine if the page footer will be displayed.
+ * @param? contextualFooter Used to determine if the page contextual band will be displayed.
```
[the remark this answers is not in the document]
Comment (ahmad-shahid) [reply src/gcweb/footer.ejs]: done

File: src/gcweb/footer.ejs [hunk 4]
```diff
@@ -4,46 +4,91 @@
...
-                <nav class="col-md-9 col-lg-10 ftr-urlt-lnk" aria-labelledby="aboutSite">
-                    <h2 class="wb-inv" id="aboutSite"><^- msg('About this site') ^></h2>
-                    <%_ if (pr.showFooter != false) { -%>
-                        <ul>
+    <h2 class="wb-inv">About this site</h2>
+    <%_ if (pr.contextualFooter != null) { -%>
+        <div class="gc-contextual">
+            <div class="container">
+                <nav>
+                    <h3><%- pr.contextualFooter.title %></h3>
```
[the remark this answers is not in the document]
Comment (ahmad-shahid) [reply src/gcweb/footer.ejs]: Updated this one and found one other place that required the change.

File: src/gcweb/footer.ejs (hunk 1 above)
Comment (ahmad-shahid) [reply src/gcweb/footer.ejs]: done

File: CHANGELOG.md (hunk 2 above)
Comment (ahmad-shahid) [reply CHANGELOG.md:1]: done

File: src/gcweb/footer.ejs [hunk 5]
```diff
@@ -4,46 +4,93 @@
...
-        <%_ } -%>
-    </div>
-    <div class="brand">
-        <div class="container">
-            <div class="row">
-                <nav class="col-md-9 col-lg-10 ftr-urlt-lnk" aria-labelledby="aboutSite">
-                    <h2 class="wb-inv" id="aboutSite"><^- msg('About this site') ^></h2>
-                    <%_ if (pr.showFooter != false) { -%>
-                        <ul>
+    <h2 class="wb-inv">About this site</h2>
```
Comment (plup2) [inline src/gcweb/footer.ejs:12]: This applies here (line 12), line 73 and line 118: Untranslated English text (line 118 used to have translation :) )
**I tried to look carefully, but double-check to make sure I didn't mess any**

Comment (ahmad-shahid) [reply src/gcweb/footer.ejs]: done

**Review by plup2 — APPROVED** [review]

**Pull request merged**